spark/spark-spa/src/main/resources/velocity/layouts/component_layout.vm

```diff
@@ -0,0 +1,51 @@
+#*
+ * Copyright 2018 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *#
+<!DOCTYPE html>
+<html class="no-js">
```

Comment:
`no-js` was added for modernizr (included transitively from zurb foundation) to know that JS has not been loaded so it can do its initial detection. See [this post](https://stackoverflow.com/questions/6724515/what-is-the-purpose-of-the-html-no-js-class). If we're not using modernizr, I suggest we remove this.